code/game/gamemodes/blob/theblob.dm

```diff
@@ -87,7 +88,10 @@
 		update_icon()
 
 /obj/effect/blob/proc/Life()
-	//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""])  \\/obj/effect/blob/proc/Life() called tick#: [world.time]")
+	if(stun_time)

```

Comment:
Checks like this should generally be ```if(stun_time > 0)``` to avoid potential issues in case it goes negative